dtfs-central-api/src/v1/controllers/tfm/deal/tfm-get-deals.controller.js

```diff
@@ -2,63 +2,32 @@ const moment = require('moment');
 const escapeStringRegexp = require('escape-string-regexp');
 const db = require('../../../../drivers/db-client');
 const CONSTANTS = require('../../../../constants');
-const getObjectPropertyValueFromStringPath = require('../../../../utils/getObjectPropertyValueFromStringPath');
-const mapDataModel = require('../../../../mapping/mapDataModel');
-const setEmptyIfNull = require('../../../../utils/setEmptyIfNull');
 const {
   isTimestampField,
   dayStartAndEndTimestamps,
 } = require('./tfm-get-deals-date-helpers');
+const { getBSSProperty } = require('../../../../mapping/mapDataModel');
 
-const sortDeals = (deals, sortBy) =>
-  deals.sort((xDeal, yDeal) => {
-    const xField = setEmptyIfNull(
-      getObjectPropertyValueFromStringPath(
-        xDeal,
-        mapDataModel(xDeal, sortBy.field),
-      ),
-    );
-
-    const yField = setEmptyIfNull(
-      getObjectPropertyValueFromStringPath(
-        yDeal,
-        mapDataModel(yDeal, sortBy.field),
-      ),
-    );
-
-    if (sortBy.order === CONSTANTS.DEALS.SORT_BY.ASCENDING) {
-      if (xField > yField) {
-        return 1;
-      }
-
-      if (yField > xField) {
-        return -1;
-      }
-    }
-
-    if (sortBy.order === CONSTANTS.DEALS.SORT_BY.DESCENDING) {
-      if (xField > yField) {
-        return -1;
-      }
+const findDeals = async (queryParameters) => {
+  const dealsCollection = await db.getCollection('tfm-deals');
 
-      if (yField > xField) {
-        return 1;
-      }
-    }
+  const {
```

Comment:
do we want to default pageSize to something as well if it's not provided here? Probably to whatever we're expecting at the moment to be the pagesize

Reply:
The problem with defaulting `pagesize` to something here is that the requirement of this ticket was to introduce these back-end pagination changes optionally and without breaking existing behaviour. Currently, if `pagesize` is not provided then we do not paginate at all, but if we defaulted `pagesize` here then we would enforce pagination.

Reply:
Ah I see, might be something we want to do on frontend then as I'd assume we always want it going forwards once we've implemented it! Sounds good